src/index.ts

```diff
@@ -69,9 +70,17 @@ import { initializeSentry, patchMatrixClient } from "./utils";
         const storage = new SimpleFsStorageProvider(path.join(storagePath, "bot.json"));
 
         let client: MatrixClient;
-        if (config.pantalaimon.use) {
+        if (config.pantalaimon.use && !config.experimentalRustCrypto) {
             const pantalaimon = new PantalaimonClient(config.homeserverUrl, storage);
             client = await pantalaimon.createClientWithCredentials(config.pantalaimon.username, config.pantalaimon.password);
+        } else if (config.experimentalRustCrypto) {
+            if (config.pantalaimon.use) {
+                console.warn("You have a pantalaimon config activated and experimentalRustCrypto. Make sure the accessToken is set and pantalaimon is disabled!");
```

Comment:
Why should we only warn? What would be the use case for saying use pantalaimon but we're gonna silently not use it and use matrix-sdk-crypto-nodejs?

Reply:
My reasoning was that this might have just been a leftover setting which doesnt change the outcome. So its more a "hey maybe cleanup your config" warning.

Reply:
I think we should refuse to start if they do that because they could not see the warning